AcademyResidentInformationApi/V1/Infrastructure/Migrations/20200915085446_AddInitialIndexes.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace AcademyResidentInformationApi.Migrations
```

Comment:
Just wondering if this file is created automatically?

Reply:
You can generate the structure of the file automatically but then I added in the SQL code to create the indexes.

Reply:
So just the bits inside the methods